DataFormats/CTPPSDigi/src/classes.h

```diff
@@ -14,6 +14,7 @@
 #include "DataFormats/CTPPSDigi/interface/TotemTriggerCounters.h"
 #include "DataFormats/CTPPSDigi/interface/TotemVFATStatus.h"
 #include "DataFormats/CTPPSDigi/interface/TotemFEDInfo.h"
+#include "DataFormats/CTPPSDigi/interface/CTPPSDiamondDigi.h"
 
 #include <vector>
 
```

Comment:
It could be a good opportunity to use here ```namespace DataFormats_CTPPSDigi``` instead of  ```namespace  DataFormats_DataFormats``` 
as used in 91X already.
I can't recall if this ever introduced an issue with duplicate symbols